src/Gui/SoTextLabel.cpp

```diff
@@ -68,31 +68,6 @@
 
 using namespace Gui;
 
-/*!

```

Comment:
Why removing the documentation?

Reply:
I don't see this as documentation but of a code commenting-out. To me the removal looks OK.

What does this document?

Reply:
What? It's a doxygen documentation on how to use the class.

Reply:
I saw the \code \endcode but don't see that it is actually describing someting. There are some code lines but no explanation what they mean or how one should handle what. For example:

`Translation { translation 4 0 0 }`

What is translated, what is the 4 about, what are the 3 numbers for...

Bot of course fine to keep it. However, it should be improved if possible that it is clear to noobs like me ;-)